src/main/java/seedu/duke/atomichabit/command/CommandResult.java

```diff
@@ -0,0 +1,25 @@
+package seedu.duke.atomichabit.command;
+
+public class CommandResult {
+    private final String feedbackToUser;
+
+    /**
+     * Constructor to be executed when class is initialized.
+     *
+     * @param feedbackToUser feedback to be printed to user
```

Comment:
Here there is a new line between description and parameters, but previous comments/files dont have. Can always keep a new line for consistency.